src/assets/wise5/common/user/user.ts

```diff
@@ -0,0 +1,10 @@
+export function usernameComparator(object1: { username: string }, object2: { username: string }) {
+  const username1 = object1.username.toLowerCase();
+  const username2 = object2.username.toLowerCase();
+  if (username1 < username2) {
+    return -1;
+  } else if (username1 > username2) {
+    return 1;
+  }
+  return 0;
```

Comment:
Can you use localeCompare() here?
```suggestion
  return username1.localeCompare(username2);
```